WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -1518,6 +1643,11 @@ templates.class.PRIEST = {
     [7] = {
       title = L["PvP Talents"],
       args = {
+        { spell = 197871, type="ability", pvptalent = 5, titleSuffix = L["cooldown"]},-- Dark Archangel
```

Comment:
you can add `buff = true` to lines with `type="ability"` to give more choices

Reply:
I know there are a bunch of things I could have done more for pvp talents, but I already spent far too much on this feature. And there are lots of small issues in there, as I didn't test them all.